types/ldapjs/index.d.ts

```diff
@@ -601,4 +601,67 @@ export class SearchEntry extends LDAPMessage {
     readonly raw: SearchEntryRaw;
 }
 
-export function parseDN(dn: string): any;
+export function parseDN(dn: string): dn.DN;
```

Comment:
According to [index.js](https://github.com/ldapjs/node-ldapjs/blob/v2.2.3/lib/index.js)
`DN` and `RDN` should also re-exported as top-level exports:
```js
module.exports = {
  ...
  dn: dn,
  DN: dn.DN,
  RDN: dn.RDN,
  parseDN: dn.parse,
  ...
```